README.md

```diff
@@ -0,0 +1,10 @@
+# SpikesLib - Examples
+Examples of how to use the [SpikesLib](https://github.com/Spikes-2212-Programming-Guild/SpikesLib2) library.
+
+## Includes:
+- Several subsystems and a drivetrain each with its own command examples.
+- Each subsystem has a RobotMap class, which holds the ports for the robot's devices.
+  - Each interface in RobotMap represents a different form of connection.
```

Comment:
is `form of connection` the right term?

Reply:
probably not but nobody in here has got a better idea

Reply:
`a different port`